specification/common-types/resource-management/v5/mobo.json

```diff
@@ -0,0 +1,37 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "5.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "MoboConfiguration": {
+      "type": "object",
+      "description": "Managed On Behalf Of configuration properties. This configuration exists for the resources where the Resource Provider manages some resources on behalf of the user.",
+      "properties": {
+        "moboBrokerResources": {
+          "type": "array",
+          "description": "Managed On Behalf Of broker resources",
+          "items": {
+            "$ref": "#/definitions/MoboBrokerResource"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ]
+        }
+      },
+      "readOnly": true
+    },
+    "MoboBrokerResource": {
```

Comment:
Same, can we avoid Mobo acronym?

Reply:
This is already approved and agreed upon resource type name which is already there in API specs.

Reply:
In this context it is NOT a resource type name. It is a schema definition. It will show up, most likely, in generated SDK codes, and this REST API doc, but its not part of the resource path, if that is what you are wondering.

Reply:
i.e. I'm not asking you to change the resoure type name in the context of this PR - although I do have the exact same worries about it!

Reply:
Got it. Thanks Tim for the review. Just putting the conversation here which we synced offline. This is the agreed and approved upon name internally and PMs. And, this is the first MOBO platform feature we are coming up with. So, I am just trying to put it so that when we get more MOBO features, we can reuse this.